src/js/angular/core/directives/rdfresourcesearch/templates/rdfResourceSearchTemplate.html

```diff
@@ -1,13 +1,15 @@
 <link href="css/rdf-resource-search.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
 
 <div ng-if="getActiveRepository() && !isLoadingLocation() && hasActiveLocation()" ng-cloak>
-    <div class="card search-rdf-input">
+    <div id="search-box" class="card search-rdf-input" ng-keydown="onKeyDown($event)">
```

Comment:
If you expect this component to be found more than once on a page, then you should avoid using IDs.

Reply:
I don't expect to see two of this directive in one view.